### ISSUE TITLE
Pagination issue

### ISSUE THREAD

**Issue by AntonioCS** [sumatrapdfreader/sumatrapdf#1086]
Hey guys,

On my laptop I have Sumatra pdf and on my android device I have adobe pdf reader.
Their seems to be a mismatch between how the pagination is done.

If I am on page 111 on sumatra it is not the same thing on adobe pdf reader.
This is a bit frustrating as I am trying to read this pdf on my laptop when I am home and on my android device when I am commuting to work.

Is their anyway to somehow make sure I get the correct page? I can see that sumatra offers 2 pages and it seems to me that adobe reader is using the second pagination, so:
```
X (Y / Z)
```

Sumatra is using X while adobe reader is using Y.
Can I set the Y of Sumatra?

**Comment by GitHubRulesOK** [comment]
Short answer is no,
       However they should in effect be the same as the Adobe standard requires in the words of the renowned Orwell "all pages are equal but some are more equal than others" (by that I mean ALL pages are +1) so the first page is counted base 0 and the reader/viewer MUST show that as 1 etc. 
The first issue is how Authors can change the lable so the first few pages may be I II III IV in those circumstances the numbering can go out the window. there is a second issue that when your rolling from one page to the next when does the page number change SumatraPDF changes when the page reaches mid screen thus 1/2 of page 1 is showing until 1/2 of page 2 gets the majority of screen when the counter then flips to page 2, I think acrobat may peak a bit earlier but I don't have it active to test.
So in summary it is possible to see a page difference if we are scrolling pages (or taken there by hyper refs) when up-to 1/2 a page is showing and may cause a 0.5 > 1.001 counter issue unless we switch to page mode.

 Going back to look at the first issue which is caused by the authors using odd numbering we need a file to see how it is conforming to Adobes requirement and if it matches the issue raised by Incorrect page numbering  #353 then it should be added as another example to that thread and this one be closed.

**Comment by GitHubRulesOK** [comment]
Without the sample to compare this issue cannot progress
